wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -75,77 +43,72 @@ export default function ShopModalView(props: ShopModelViewProps) {
         onChange={props.handleImageSelection}
       />
       <div className="col-6">
-        {/* Test 1 */}
         <WavyModalHeader image={props.headerPictureUrl || defaultHeaderPicture} onClose={props.onClose}>
```

Comment:
Ts wants to to define what type of data the URL will be up in ShopModelViewProps